bcftbx/cmdparse.py

```diff
@@ -157,33 +157,34 @@ def parse_args(self,argv=None):
         except KeyError:
             # No parser
             self.error("Usage: %s COMMAND [options] [args...]\n\n"
-                       "%s: error: no such command: %s" % 
-                       (self._name,self._name,cmd))
+                       "%s: error: no such command: %s" %
+                       (self._name, self._name, cmd))
+
         # Parse the remaining arguments and return
-        options,arguments = p.parse_args(argv[1:])
-        return (cmd,options,arguments)
+        options, arguments = p.parse_args(argv[1:])
```

Comment:
If using `argparse.ArgumentParser` then the `parse_args` call here will not return enough values to unpack (it returns a single `Namespace` object, instead of the `(options,arguments)` tuple returned by `optparse.OptionParser.parse_args`.

Reply:
So, probably we can change it to [`parse_known_args`](https://docs.python.org/3/library/argparse.html#partial-parsing)